docs/variants/protectli_vp2420/releases.md

```diff
@@ -0,0 +1,70 @@
+# Release Notes
+
+Following Release Notes describe status of open-source firmware development for
+Protectli VP2420 family.
+
+For details about our release process please read
+[Dasharo Standard Release Process](../../dev-proc/standard-release-process.md).
+
+<center>
+[Subscribe to Protectli VP2420 Dasharo Release Newsletter]
+[newsletter]{.md-button .md-button--primary .center}
+</center>
+
+Test results for this platform can be found
+[here](https://docs.google.com/spreadsheets/d/1wI0qBSLdaluayYsm_lIa9iJ9LnPnCOZ9eNOyrKSc-j4/edit#gid=975611333).
+
+## v1.0.0 - 2022-12-22
+
+### Added
+
+- description: Support for VP2420 platform
```

Comment:
This is YAML format, not markdown, use the dasharo/dasharo script to generate releases.md file and paste here the v1.0.0 section